templates/todo/projects/csharp-sql-swa-func/.gitignore

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
You can delete this file.  It will be added to the repo via repoman.

Reply:
Deleted